main.tf

```diff
@@ -15,7 +15,8 @@
  */
 
 locals {
-  tables = { for table in var.tables : table["table_id"] => table }
+  tables = { for table in var.tables : table["table_id"] => table if lookup(table, "view", null) == null }
+  views  = { for table in var.tables : table["table_id"] => table if lookup(table, "view", null) != null }
 }
 
 resource "google_bigquery_dataset" "main" {
```

Comment:
We also need to update the dataset to support [views](https://www.terraform.io/docs/providers/google/r/bigquery_dataset.html#view) as requested in #41.